Cargo.toml

```diff
@@ -1,3 +1,5 @@
+[workspace]
```

Comment:
Why is a workspace token added? This is not a workspace.

Reply:
I cloned this repo into my workspace to edit locally and wasn't able to run `cargo fmt` on only this crate since it is obviously not part of my workspace.

Reply:
Makes sense, please remove from this repo.